README.md

```diff
@@ -225,4 +220,25 @@ Each directory will be created with `cl-keytool.jar`, `cl-wallet.jar`, and a P12
 **In `code/data-l1`:**
 -   data-l1.jar     // The executable for the dL1 layer
 
+
+### `hydra remote_start`
+
+This method initiates the remote startup of your metagraph in one of the available networks: testnet, integrationnet, and mainnet.
+
+To commence the remote startup of the metagraph, we utilize the parameters configured on Euclid. The startup process unfolds as follows:
```

Comment:
Which parameters configured in Euclid do you mean here? Maybe it could be more specific. 

```suggestion
To begin the remote startup of the metagraph, we utilize the parameters configured in Euclid. The startup process unfolds as follows:
```

Reply:
Got it, I'll add